docs/modules/globby.md

```diff
@@ -0,0 +1,46 @@
+# globby
+
+`globby` is a globbing library that presents [some behaviors](https://github.com/sindresorhus/globby#features)
+no other libraries have, mainly the handling of negated patterns and directory expansion. It can be replaced
+with lighter and faster alternatives.
+
+# Alternatives
+
+## tinyglobby
+
+`tinyglobby` can be used as an alternative in most cases.
+It doesn't include all `globby` options though.
+
+[Project Page](https://github.com/SuperchupuDev/tinyglobby)
+
+[npm](https://npmjs.com/package/tinyglobby)
+
+## fdir + picomatch
+
+If you don't rely on `globby`'s special features, you can just use
+`fdir` combined with `picomatch`.
+Its API is nothing like `globby`'s, but it does the job if configured properly.
+
+[`fdir` Project Page](https://github.com/thecodrr/fdir)
+
+[`fdir` on npm](https://npmjs.com/package/fdir)
+
+[`picomatch` Project Page](https://github.com/micromatch/picomatch)
+
+[`picomatch` on npm](https://npmjs.com/package/picomatch)
+
+## Node.js (since v22.x)
+
+Node.js v22 and up supports the [`fs.glob`](https://nodejs.org/api/fs.html#fspromisesglobpattern-options)
+function. Although it's (currently) not as fast as other alternatives, it works as a zero-dependency
+solution. It doesn't have `globby`'s special features though.
```

Comment:
i don't think we need the last sentence about globby features

Reply:
how else would readers know about compatibility aspects between the two? some could think it covers all globby features

Reply:
because what exactly are globby's "special features"? from this text, that isn't clear. and if you make it clear, you're going to basically duplicate globby's readme

what is the common use case of globby? if it is to glob patterns, regardless of whatever niche options they have, we probably don't need to care so much about the options

Reply:
it's not about its options but rather about its default behavior, like autoconverting i.e a `src` pattern to a `src/**/*` one. i'd say that's the whole point of globby - globbing with added behavior that can be considered useful. users of libraries that blindly switch to alternatives without taking that into account will end up seeing breaking behavior if those patterns are user provided (in fact that's what happened in tsup after me forgetting to add some globby behavior before it was suggested to move that logic into tinyglobby)

Reply:
if some globby-specific feature is that important, we should probably be defining exactly which feature is missing in each of these replacements

nobody knows what globby's "special features" are. we need to be exact here and say specifically what isn't supported

Reply:
thought the lines all the way to the top were clear enough, will do 👍